build/yamls/patches/release/.gitignore

```diff
@@ -1 +0,0 @@
-# placeholder
```

Comment:
not release with this PR change, but I saw there are a few other `.gitignore` files have the same `#placeholder`, do we need them? eg: build/yamls/flow-visibility/patches/release/.gitignore.